### PR TITLE
Wrote events for LUG after the takeover.

### PR DESCRIPTION
**Pull request by ckeltz**
Not quite done yet, but this is what I have so far.

### CODE REVIEW

**Comment by ckeltz** [comment]
@magical Need your input on Week 4.

File: content/events/2016/fall2016/20161101_wk6.mkd [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+preview: Come argue for or against your favorite Linux tools!
 location: Kelly Engineering Center 1007
 
 ---
 
-Come to a meeting or [make an issue on the GitHub repo for this site][issuses]
-to suggest an event for this LUG meeting!
-
-[issues]: https://github.com/OSULUG/OSULUG-Website/issues
+Tonight we'll be debating over what's best. Vi(m) or emacs? Firefox or Chrome?
```
Comment (aaroncohen73) [inline content/events/2016/fall2016/20161101_wk6.mkd]: Don't forget tabs vs spaces!

File: content/events/2016/fall2016/20161129_wk10.mkd [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
 ---
 
-Come to a meeting or [make an issue on the GitHub repo for this site][issuses]
-to suggest an event for this LUG meeting!
-
-[issues]: https://github.com/OSULUG/OSULUG-Website/issues
+School is always stressful towards the end of the term, and at few points is this
+more apparent than "Dead Week". Come take a break with us, we'll have board games,
+card games, and maybe snacks! Just keep the Cheeto fingers off the game pieces
```
Comment (aaroncohen73) [inline content/events/2016/fall2016/20161129_wk10.mkd]: Should we also invite people to bring their own board/card games?

File: content/events/2016/fall2016/20161108_wk7.mkd [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
+preview: We will be going over the basics of HTTP and REST so you can start using them in your programs.
 location: Kelly Engineering Center 1007
 
 ---
 
-Come to a meeting or [make an issue on the GitHub repo for this site][issuses]
-to suggest an event for this LUG meeting!
-
-[issues]: https://github.com/OSULUG/OSULUG-Website/issues
+Ever been confused about how programs actually access the internet? Ever wonder how
```
Comment (aaroncohen73) [inline content/events/2016/fall2016/20161108_wk7.mkd]: I think that the first sentence should be replaced with "Ever been confused about how programs talk to each other over the Internet?" and the second sentence could be removed entirely in my opinion. +1 for the RFC 1149 reference though

Comment (ckeltz) [inline content/events/2016/fall2016/20161108_wk7.mkd]: Also RFC 2324! Yeah, I'll fix the first couple sentences, thanks for the input

**Pull request merged**